docs/projects/cn0577/index.rst

```diff
@@ -0,0 +1,294 @@
+
+-  The clock architecture of the :adi:`EVALCN0577-FMCZ <CN0577>` is designed
+   with careful consideration to ensure low jitter and low phase noise
+
+-  An on-board 120 MHz voltage controlled crystal oscillator (VCXO) is used to
+   provide the clock for the :adi:`EVALCN0577-FMCZ <CN0577>` board and the FPGA.
+   It is further named as reference clock. This clock is gated and fed back to
+   the device as the sampling clock, on which the data was sampled
+
+-  The DMA runs on the ZynqPS clock FCLK_CLK0 which has a frequency of 100MHz
+
+For more detalis, check :adi:`EVALCN0577-FMCZ <CN0577>` datasheet.
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL (see more at :ref:`architecture`).
+
+==================== ===============
+Instance             Zynq/Microblaze
+==================== ===============
+axi_ltc2387          0x44A0_0000
+axi_ltc2387_dma      0x44A3_0000
+axi_pwm_gen          0x44A6_0000
+==================== ===============
+
+I2C connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 20 20 20 20 20
+   :header-rows: 1
+
+   * - I2C type
+     - I2C manager instance
+     - Alias
+     - Address
+     - I2C subordinate
+   * - PL
+     - iic_fmc
+     - axi_iic_fmc
+     - 0x4162_0000
+     - ---
+   * - PL
+     - iic_main
+     - axi_iic_main
+     - 0x4160_0000
+     - ---
+
+GPIOs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 20 20 20 15
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+     - Zynq MP
+   * - pd_cntrl
+     - IN
+     - 32
+     - 86
+     - 110
+   * - testpat_cntrl
+     - IN
+     - 33
+     - 87
+     - 111
+
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+================ === ========== ===========
+Instance name    HDL Linux Zynq Actual Zynq
+================ === ========== ===========
+ad_cpu_interrupt 13  57         89
+================ === ========== ===========
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here <>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0577/zed
+   user@analog:~/hdl/projects/cn0577/zed$ make
```

Comment:
Give an example on how to use the parameter:
```suggestion
   user@analog:~/hdl/projects/cn0577/zed$ make TWOLANES=0
```

Reply:
Should I add this as a second example? Or follow the example as it is in AD4630-FMC documentation?

Reply:
Ok, add it as a second example, and specify like a comment, that when running "make", the default configuration is ...